shared/user.js

```diff
@@ -11,13 +11,13 @@
   User.PASSWORD_MIN_LENGTH = 8;
 
   User.permissions = {
-    'view data': ['viewer', 'monitor', 'manager', 'admin'],
-    'edit incidents': ['monitor', 'manager', 'admin'],
-    'edit reports': ['monitor', 'manager', 'admin'],
-    'edit sources': ['manager', 'admin'],
-    'toggle fetching': ['manager', 'admin'],
-    'manage trends': ['manager', 'admin'],
+    'view data': ['viewer', 'monitor', 'admin'],
```

Comment:
If we were doing this over again I would have suggested not removing the granularity (incidents/reports/sources) in favour of a more generic 'data', because it handcuffs us down the line if we want to tweak permissions. If you think that could be reinstated without too much effort I would do it, but this is probably okay also for now.